examples/vision/barlow_twins.py

```diff
@@ -959,7 +967,9 @@ def train_step(self, batch: tf.Tensor) -> tf.Tensor:
 # chose the LAMB optimizer due to high batch sizes. Converged MUCH faster
 # than ADAM or SGD
 optimizer = tfa.optimizers.LAMB()
-loss = BarlowLoss(BATCH_SIZE)
+
+# We can just drop in either one of the two(results will be similar for both)
+loss = tensorflow_similarity.losses.Barlow()  # BarlowLoss(BATCH_SIZE)
```

Comment:
What is the purpose of the comment `# BarlowLoss(BATCH_SIZE)`?

Reply:
So that people can just drop in the from-scratch version and switch between the two.

Reply:
(sorry i think i responded somewhere else but i'm not seeing it)

So that people can switch between the two easily.